addons/jest/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@storybook/addon-jest",
+  "version": "3.2.14",
+  "description": "React storybook addon that show component jest report",
+  "keywords": [
+    "addon",
+    "jest",
+    "react",
+    "report",
+    "results",
+    "storybook",
+    "unit-testing"
+  ],
+  "homepage": "https://storybook.js.org",
+  "bugs": "https://github.com/storybooks/storybook",
+  "license": "MIT",
+  "author": "Renaud Tertrais <renaud.tertrais@gmail.com> (https://github.com/renaudtertrais)",
+  "files": [
+    "dist",
+    "styles.js",
+    "register.js"
+  ],
+  "main": "dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/storybooks/storybook"
+  },
+  "scripts": {
+    "prebuild": "npm run clear",
+    "build": "cross-env NODE_ENV=production babel -d ./dist ./src",
```

Comment:
We need a `prepare` script instead (see other addons). All the babel-related stuff can be removed from package as we use top-level babel dependencies and setup for transpiling everything